src/components/input-group/daikin-input-group.ts

```diff
@@ -135,30 +147,47 @@ export class DaikinInputGroup extends LitElement {
   }
 
   override render() {
-    const inputGroupLabelClassName = cvaLabel({
-      variant: this.disabled ? "disabled" : "enabled",
-      required: this.required ? "required" : "optional",
-    });
+    const isHelper = !!this.helper.length && !this.error.length;
+    const isError = !this.disabled && !!this.error.length;
 
-    const inputGroupHelperClassName = cvaHelper({
-      variant: this.disabled ? "disabled" : "enabled",
-    });
+    console.log(isHelper, isError);
 
     return html`<fieldset class="content" ?disabled=${this.disabled}>
-      <label class="flex flex-col justify-center w-max gap-1 font-daikinSerif">
-        ${this.label
-          ? html`<span class=${inputGroupLabelClassName}>${this.label}</span>`
-          : null}
+      <label class="flex flex-col justify-center gap-2 w-max font-daikinSerif">
+        <span
+          class=${cvaLabel({
+            required: this.required,
+          })}
+        >
+          ${this.label}
+        </span>
         <slot @slotchange=${this._handleSlotChange}></slot>
-        ${this.helper && !this.error
-          ? html`<span class=${inputGroupHelperClassName}>${this.helper}</span>`
-          : null}
-        ${!this.disabled && !!this.error
-          ? html`<span
-              class="flex gap-2 text-[--input-group-border-color-error] leading-[22px] before:i-daikin-status-negative before:block before:w-[16px] before:h-[22px]"
-              >${this.error}</span
-            >`
-          : null}
+        <div
+          class=${cvaHelperAndErrorContainer({
+            visible: isHelper || isError,
+          })}
+        >
+          ${isHelper
+            ? html`<span
+                class=${cvaHelper({
+                  disabled: this.disabled,
+                })}
+              >
+                ${this.helper}
+              </span>`
+            : nothing}
+          <div class="flex items-center gap-1 h-max">
+            ${isError
+              ? html`<daikin-icon icon="error"></daikin-icon>`
+              : nothing}
+            <span
+              class="text-[--input-group-border-color-error] text-sm font-bold leading-5"
+              aria-live="polite"
+            >
+              ${isError ? this.error : ""}
+            </span>
+          </div>
+        </div>
```

Comment:
Can't we do like following?

```suggestion
        <div
          class=${cvaHelper({
            type: helperType, /* "helper" | "helperDisabled" | "error" | "none" */
          })}
          aria-live=${helperType === "error" ? "polite" : nothing}
        >
          ${helperText}
        </div>
```